app/views/layouts/mailer.html.erb

```diff
@@ -25,6 +25,49 @@
         font-weight: normal;
         margin-bottom: 2.5rem;
       }
+      .email-contents button {
+          background-color: #121111;
+          border-bottom: #121111 2px solid;
+          border-bottom-left-radius: 10px;
+          border-bottom-right-radius: 10px;
+          border-image-outset: 0;
+          border-image-repeat: stretch;
+          border-image-slice: 100%;
+          border-image-source: none;
+          border-width: 2px;
+          border-image-width: 1;
+          border-left-color: #121111;
+          border-left-style: solid;
+          border-right-color: #121111;
+          border-right-style: solid;
+          border-top-color: #121111;
+          border-top-left-radius: 10px;
+          border-top-right-radius: 10px;
```

Comment:
could we also replace `border-bottom-left-radius`, `border-bottom-right-radius`, `border-top-left-radius` and `border-top-right-radius` with:
```suggestion
          border-radius: 10px;
```